Dockerfile

```diff
@@ -7,16 +7,49 @@ RUN apt-get update && apt-get install -y \
   mediainfo \
   unzip \
   && rm -rf /var/lib/apt/lists/*
-
 RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
 RUN php composer-setup.php --install-dir=/bin --filename=composer --version=1.10.16
 RUN php -r "unlink('composer-setup.php');"
```

Comment:
🙌 this is looking good. A few minor questions that might DRY things up.

Looking at https://github.com/docker-library/drupal/blob/b51cf402be28ed417827779e023a04da1888e8b0/8.9/apache-buster/Dockerfile#L66-L73 they have a nice one-step install/clear-out of Composer which might reduce our image step size. I know this isn't a change in this PR so it's not necessary for this one, but it's a good example of squashing install & cleanup steps into a single build step.